### PR TITLE
Updated rule names for consistency and corrected small grammar typos

### PR DESCRIPTION
**Pull request by vkresch**
#### Reference to a related issue in the repository
This PR is more of a refactoring and correcting grammar mistakes in the comments.

#### Add a description
This PR updates the rule naming used in the osi-validator to a more intuitive way (like `is_equal` --> `is_equal_to` ). It adds also an option to run the validator either sequentially or in parallel with the -p flag and corrects some issues referred [here](https://github.com/OpenSimulationInterface/osi-documentation/issues/3).

#### Mention a member
@jdsika pls review and approve like always :)

#### Check the checklist

- [x] My code and comments follow the [contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/howtocontribute.html) of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the [documentation](https://github.com/OpenSimulationInterface/osi-documentation) for osi-validation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests / travis ci pass locally with my changes.

### CODE REVIEW

**Comment by jdsika** [comment]
The spelling in the comments is horrific

File: osivalidator/osi_general_validator.py [hunk 1]
```diff
@@ -126,16 +130,19 @@ def main():
...
-            process_timestep(i)
-        # pool.map(process_timestep, range(first_of_blast, last_of_blast))
-
-        LOGGER.flush(LOGS)
-
+        if args.parallel:
+            # Launch parallel computation
+            # Recreate the pool
+            pool = Pool()
+            pool.map(process_timestep, range(first_of_blast, last_of_blast))
```
Comment (jdsika) [inline osivalidator/osi_general_validator.py]: How is an abort of the program handled here?

Comment (vkresch) [reply osivalidator/osi_general_validator.py]: You mean how it is closed or how to handle when an exception occurs? There is a `close_pool` function call for terminating after the line `pool.map` (see [here](https://github.com/OpenSimulationInterface/osi-validation/blob/rule-naming-update/osivalidator/osi_general_validator.py#L138)). For exception handling there needs to be implemented a try catch if something goes wrong during processing.

Comment (jdsika) [reply osivalidator/osi_general_validator.py]: Ok, fine thank you!

**Pull request merged**